### PR TITLE
Unexpected behaviour in the scholarX FAQ section

### LINKED ISSUE #407: Unexpected behaviour in the scholarX FAQ section
**Describe the bug**
The scholarX page's FAQ sections are not working properly.
 1. `How to be a mentor` section
   - Some unwanted characters are displaying. (`aria-controls...`)
   - Sections are not expanding properly
Screenshot 2020-01-18 at 08 10 28
2. `How to be a mentee?` section
   - Each divs should have a unique id and should have linked with appropriate data-target
     (currently, every divs has the same id named `collapseTwo` also, headings have the same id named `headingTwo`. Replace them with unique ids) 

**To Reproduce**
1. Go to `scholarX.html`
3. Scroll down to 'How to be a mentor' section

**Additional context**

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #407 

## Goals

- Unexpected behaviour in the scholarX FAQ section

## Approach

- Remove unwanted characters
- Replace them with unique ids

### Screenshots

Screenshot from 2020-01-18 12-17-15

  
### Preview Link

https://pr-410-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
- There are repeated IDs on this page. Please replace them with unique ids.
- Change your commit message's title.

**Comment by anjula-sack** [comment]
I'm working on this.

File: src/main/webapp/scholarx.html [hunk 1]
```diff
@@ -316,10 +237,10 @@ <h5 class="mb-0">
                                             </h5>
                                         </div>
                                         <div
-                                                id="collapseTwo"
+                                                id="collapseFive"
                                                 class="collapse"
-                                                aria-labelledby="headingTwo"
-                                                data-parent="#accordionExample"
+                                                aria-labelledby="headingFive"
+                                                data-parent="#accordionExampleFive"
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx.html]: ` data-parent="#accordionExampleTwo"`

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

**Review by jayasanka-sack — APPROVED** [review]

**Comment by jayasanka-sack** [comment]
Good job!

**Pull request merged**